pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/Context.java

```diff
@@ -40,15 +36,8 @@
  * executing function
  */
 @InterfaceAudience.Public
-@InterfaceStability.Stable
-public interface Context {
-    /**
-     * Access the record associated with the current input value.
-     *
-     * @return
-     */
-    Record<?> getCurrentRecord();
-
+@InterfaceStability.Unstable
```

Comment:
why are changing the InterfaceStability level?  Is this change no backwards compatible?